charts/dvpe-deployment-gloo/templates/deployment.yaml

```diff
@@ -17,6 +23,11 @@ spec:
     metadata:
       labels:
         app: {{ $serviceName }}
+        {{- if .Values.datadog.enabled }}
+        tags.datadoghq.com/env: {{ .Values.datadog.env }}
+        tags.datadoghq.com/service: {{ $serviceName }}
+        tags.datadoghq.com/version: {{ if .Values.datadog.version }}{{- .Values.datadog.version }}{{- else }}{{- .Values.deployment.spec.image.tag }}{{- end }}
```

Comment:
why give the possibility to override the image tag version at all and not just use it always?